TransferMethod/Sources/View/Widget/TextWidget.swift

```diff
@@ -260,6 +245,16 @@ class TextWidget: AbstractWidget {
         }
         return ""
     }
+
+    private func getCharacterAt(_ index: String.Index, _ text: String) -> Character {
+        let range = index ..< text.index(after: index)
+        return [Character](text[range]).first!
```

Comment:
Not a good idea to use force unwrap `!` here - kindly refactor this

Reply:
If we know `String.Index`, why don't we use string subscript? We can replace this function by `text[index]`

Reply:
remove this function at all

Reply:
@ymoskovko-epam swift doesn't support subscripting for String.. there is a lot of discussion around it online

Reply:
@ddhar-hw here index is guaranteed to be less than String.endIndex as it is being called from the `while` loop where we break the loop when current index is equal to or greater than end index, due to this range will always have a character. Think of this as if I am calling String.charAt(index) and I know index is always less than end index due to loop condition.
Also, If I were to replace `!` by `?` then I will need to provide a default character value which I can't think of any.

Reply:
@ymoskovko-epam thanks, fixed